### PR TITLE
Feature/HD-920 update base image to centos8

### PR DESCRIPTION
**Pull request by lavigne958**
Updated the base image from centos 7 to centos 8 to match the base image on the controller repository.

Added extra repo PowerTools, use a local worker for the master worker as it only trigger the stages we don't need a kub_pod.

Set golang and golint version to match what is in the controller repository.

### CODE REVIEW

File: eve/workers/compile_centos_8/Dockerfile [hunk 1]
```diff
@@ -21,13 +25,13 @@ RUN yum install -y \
...
 # Installation of golang
-ARG GO_VERSION=1.14.2
+ARG GO_VERSION=1.14
 RUN curl -s https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | \
        tar -C /usr/local -xzf - && \
     ln -snf /usr/local/go/bin/go /usr/bin
 
 # Linter
-ARG GOLINT_VERSION=1.23.8
+ARG GOLINT_VERSION=1.21.0
```
Comment (benoit-a) [inline eve/workers/compile_centos_8/Dockerfile:35]: why the revert ?

File: eve/workers/compile_centos_8/Dockerfile [hunk 2]
```diff
@@ -21,13 +25,13 @@ RUN yum install -y \
         zlib-devel
 
 # Installation of golang
-ARG GO_VERSION=1.14.2
+ARG GO_VERSION=1.14
```
Comment (benoit-a) [inline eve/workers/compile_centos_8/Dockerfile]: revert ?

Comment (lavigne958) [reply eve/workers/compile_centos_8/Dockerfile]: I used the same version as mentioned in the Dockerfile for centos8 in the repository scality/hdcontroller.

It got updated in between this PR and today. 

I suppose we should use the exact same version of goland in both repositories.

File: eve/workers/compile_centos_8/Dockerfile (hunk 1 above)
Comment (lavigne958) [reply eve/workers/compile_centos_8/Dockerfile:35]: same here, I took the version number from the repository scality/hdcontroller in the dockerfile for centos8 to have the exact same version of linter for both repositories.

**Review by lavigne958 — COMMENTED** [review]
I propose we update both repositories to the latest version, so for `GO_VERSION` we use `1.15.1` (the version that is set in the hdcontroller repo) and for the linter we use `1.23.8` (the version previously used here. This way both have the same version and both have latest version that we choose.

**Review by slaperche-scality — APPROVED** [review]

**Review by benoit-a — APPROVED** [review]

**Pull request merged**